packages/common/src/styles/RTLProvider.tsx

```diff
@@ -4,6 +4,10 @@ import { useRtl } from '@common/intl';
 export const RTLProvider: FC = props => {
   const isRtl = useRtl();
 
+  React.useLayoutEffect(() => {
```

Comment:
was looking at the arrow on autocomplete issue and one suggestion was to set the `dir` on body / containing element. don't think it helped, but since it was here I left it in 🤷